spec/features/ua_view_items_from_home_page_spec.rb

```diff
@@ -1,8 +1,14 @@
 require 'rails_helper'
 
 describe 'unauthorized user view items by item and category', type: :feature do
-  let(:item1) { Fabricate(:item) }
-  let(:item2) { Fabricate(:item) }
+  let(:item1) { item = Fabricate.build(:item)
+                item.categories << category1
+                item.save
+                item}
+  let(:item2) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.